build.gradle

```diff
@@ -161,3 +164,47 @@ void execAndFailIfTextFound(String textToFind, Object... args) {
       throw new GradleException('ANT build failure')
   }
 }
+
+
+def userHome = System.properties['user.home']
+def mpsPluginsDirPattern
+if (System.properties['os.name'].toLowerCase().contains('mac')) {
+    mpsPluginsDirPattern = "$userHome/Library/Application Support/%s"
+} else {
+    mpsPluginsDirPattern = "$userHome/.%s/config/plugins"
+}
+
+if (project.hasProperty("MPS_PATHS_SELECTOR")) {
+    ext.mpsPluginsDir = sprintf(mpsPluginsDirPattern, project.getProperty("MPS_PATHS_SELECTOR"))
+} else {
+    ext.mpsPluginsDir = sprintf(mpsPluginsDirPattern, "MPS$mpsMajor")
+}
+
+task install_git4mps(type: Copy,
+    description: "Install git4mps into the MPS plugin repository",
+	dependsOn: build) {
+    from zipTree("$rootDir/dist/build/artifacts/reviewPlugin/git4mps.zip")
+    into "$mpsPluginsDir"
+}
+
+task install_mpsreview(type: Copy,
+    description: "Install mps.review into the MPS plugin repository",
+	dependsOn: build) {
+    from zipTree("$rootDir/dist/build/artifacts/reviewPlugin/com.workday.mps.review.zip")
+    into "$mpsPluginsDir"
+}
+
+task install(description: "Install the required plugins into the MPS plugin repository by copying them into the user's home dir",
+    group: "MPS Build",
+	dependsOn: [install_git4mps, install_mpsreview]) {
+    doFirst {
+        // check parent gradle file for definition of the variables
+        println "Installing required mps-code-reviewer plugins to '$mpsPluginsDir'"
+        if (!project.hasProperty("MPS_PATHS_SELECTOR")) {
+            println "To change 'MPS<>' part, pass MPS_PATHS_SELECTOR property to gradle with -PMPS_PATHS_SELECTOR=<custom path selector>"
```

Comment:
Can you please change this part so that it is more descriptive? What does "MPS<>" part mean?